tutor/src/components/chapter-section.js

```diff
@@ -3,11 +3,11 @@ import PropTypes from 'prop-types';
 import ChapterSectionModel from '../models/chapter-section';
 
 export default
-function ChapterSection({ chapterSection }) {
+function ChapterSection({ chapterSection, hideClassName }) {
   if (!chapterSection || chapterSection.isEmpty) { return null; }
   return (
     <span
-      className="chapter-section"
+      className={hideClassName == true  ? '' : 'chapter-section'}
       data-chapter-section={chapterSection.key}
```

Comment:
This seems a bit strange to me.  Do we need to hide the class so it's not styled?  If so, lets add a string `className`  prop something like this, and I'll propose an alternative below

```js
import cn from 'classnames';

export default
function ChapterSection({ chapterSection, className }) {
  ...
  return (
    <span className={cn('chapter-section', className)}> 
    ...
  )
}

ChapterSection.propTypes = {
  className: PropTypes.string,
}